server/socket/socketManager.js

```diff
@@ -78,16 +78,16 @@ exports.socketManager = (io, socket) => {
 
```

Comment:
Oddly enough, I can't comment on lines 27 and 29. This comment is for line 27 and line 29. Please change there robot_id to robotId

Reply:
well spotted!